File: preview/understand/query/update-default/update-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

query = @client.preview.understand
                       .services('UAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                       .queries('UHXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                       .update(
                          sample_sid: 'UFXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
                        )

puts query.account_sid

File: preview/sync/sync_list_item/update-default/update-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Converters;
using Twilio.Rest.Preview.Sync.Service.SyncList;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var syncListItem = SyncListItemResource.Update(
            pathServiceSid: "ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
            pathListSid: "ESXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
            pathIndex: 1,
            data: "{}"
        );

        Console.WriteLine(syncListItem.Index);
    }
}
